src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import Redux from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = Redux.combineReducers({
+  count: booksReducer,
+  auth: categoriesReducer,
+});
+
+const store = configureStore({ reducer: rootReducer });
+
+console.log(store.count.getState());
```

Comment:
- [OPTIONAL] It is not good JavaScript practice to leave `console.log` statements in your code base. It could reveal sensitive information to people through the console of the browser. As much as the statement is good for debugging, it is not a great practice to leave them in the codebase. 👍